cmd/ocsp-responder/main.go

```diff
@@ -233,15 +225,16 @@ func (src *dbSource) Response(ctx context.Context, req *ocsp.Request) ([]byte, h
 	select {
 	case <-ctx.Done():
 		if errors.Is(ctx.Err(), context.Canceled) {
-			src.metrics.ocspLookups.WithLabelValues("canceled").Inc()
+			src.metrics.ocspLookups.WithLabelValues("mysql", "canceled").Inc()
 		}
-		src.metrics.ocspLookups.WithLabelValues("deadline_exceeded").Inc()
+		src.metrics.ocspLookups.WithLabelValues("mysql", "deadline_exceeded").Inc()
```

Comment:
Looks like we'll double-count here. Should be an if/else.

Reply:
Thanks, fixed. It looks like the only other option is deadline_exceeded, so made that assumption in a basic else.